spec/default_spec.rb

```diff
@@ -0,0 +1,69 @@
+require_relative 'spec_helper'
+
+describe 'rsc_tomcat::default' do
+  let(:chef_run) do
+    ChefSpec::ServerRunner.new do |node|
+      node.set['rsc_tomcat']['application_name'] = 'sample'
+      node.set['rsc_tomcat']['listen_port'] = '8080'
+      node.set['rsc_tomcat']['bind_network_interface'] = 'private'
+      node.set['rsc_tomcat']['vhost_path'] = 'www.example.com'
+      node.set['rsc_ros']['file'] = 'sample/sample.war'
+      node.set['rsc_ros']['provider'] = 'aws'
```

Comment:
no, that is for the rspec test.  the .kitchen.yml is what you want to look at.  see the attributes, you'll need to add some ENV entries